lib/bundler/cli/gem.rb

```diff
@@ -139,6 +139,10 @@ def run
 
       # Open gemspec in editor
       open_editor(options["edit"], target.join("#{name}.gemspec")) if options[:edit]
+    rescue SystemCallError => e
```

Comment:
@segiddins the error is thrown by `thor.template`. Should I monkey-patch it? Or how would you use `SharedHelpers.filesystem_access`?